worlds/civ_6/docs/setup_en.md

```diff
@@ -0,0 +1,51 @@
+# Setup Guide for Civilization VI Archipelago
+
+This guide is meant to help you get up and running with Civlization VI in your Archipelago run. Note that this requires you to have both Rise & Fall as well as Gathering Storm installed. This will not work unless both of those DLCs are enabled.
+
+## Requirements
+
+The following are required in order to play Civ VI in Archipelago
+
+- Windows OS (Firaxis does not support the necessary tooling for Mac, Linux is yet to bet verified)
+
+- Installed [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases) v0.4.5 or higher.\
+   **Make sure to install the Generator if you intend to generate multiworlds.**
+
+- The latest version of the [Civ VI AP Mod](https://github.com/hesto2/civilization_archipelago_mod).
+
+- Tuner setting enabled so the archipelago client can communicate with the game

```

Comment:
The "Tuner setting" line seems redundant with the section below it, you should consider removing it